### ISSUE TITLE
GRUB error message when choosing submenu in EFI mode

### ISSUE THREAD

**Issue by mika** [grml/grml#16]
```
error: null src bitmap in grub_video_bitmap_create_scaled.
Loading kernel...
Loading initrd...

Press any key to continue...
```

**Comment by mika** [comment]
@schierlm - do you have any hints for me/us where to investigate regarding this issue? Are we missing a module or is that related to wrong video configuration?

**Comment by schierlm** [comment]
Problem is caused by the theme. When the theme is disabled, the error disappears.

I have no idea whether it is some syntax error in the theme file I cannot see, or a bug in the theme engine (and I did not verify if it is yet another bug in Debian's heavily patched GRUB or also present in upstream 2.02 final), but at least if I replace the desktop-color by a desktop-image of a small black .png file, the error message disappears (Presumably also for different size background images).

**Comment by mika** [comment]
Thanks a *lot* for your help, @schierlm!
